src/prop/cnf_stream.cpp

```diff
@@ -709,11 +708,7 @@ void TseitinCnfStream::convertAndAssert(TNode node, bool negated) {
   Debug("cnf") << "convertAndAssert(" << node
                << ", negated = " << (negated ? "true" : "false") << ")" << endl;
 
-  if (d_convertAndAssertCounter % ResourceManager::getFrequencyCount() == 0) {
-    d_resourceManager->spendResource(ResourceManager::Resource::CnfStep);
-    d_convertAndAssertCounter = 0;
-  }
-  ++d_convertAndAssertCounter;
+  d_resourceManager->spendResource(ResourceManager::Resource::CnfStep);
```

Comment:
I believe that we can get rid of `d_convertAndAssertCounter` altogether.

Reply:
Ditto.